camelot/admin/action/field_action.py

```diff
@@ -177,7 +177,7 @@ def model_run(self, model_context):
         admin.add(obj)
         admin.set_defaults(obj)
         yield action_steps.UpdateEditor('new_value', obj)
-        yield action_steps.OpenFormView([obj], admin)
+        yield action_steps.OpenFormView(obj, model_context.proxy, admin)
```

Comment:
werken dezen ?  ik zou niet verwachten dat hier een model_context.proxy bestaat ?

en als die bestaat is dat mss niet die waar obj in zit ?

Reply:
Nee klopt, ik dacht dat ik ze allemaal getest had, maar deze heb ik inderdaad over het hoofd gezien.
Zie e4eb3a657485cadc55bbd9e242b7d366411341d1.